massgov/mayflower
# DP-16287 Add a new overlay color to key message

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

- Add a new green, #214F34, to the theme colors as `$c-berkshires-green-dark`. Its theme color usage variable is set as $c-primary-alt-dark.
- Set up a logic to switch `$c-primary-alt` for solid background and `$c-primary-alt-dark` as an overlay for bg image of the key message in content area.
   - When background image data is available with green background option, `$c-primary-alt-dark` is used as its overlay color.
   - When no background image data is available, `$c-primary-alt` is used as the background color.

## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-16287)

## Steps to Test

Use /patterns/05-pages-marketing-campaign/05-pages-marketing-campaign.html as a test page.
1. Go to /patternlab/styleguide/source/_patterns/05-pages/marketing-campaign.json
2. At L. 80, change the `"backgroundColor"` value to `c-primary-alt`.
3. Go to the test page and check the key message right above the footer.
4. Find it has solid green background.  The color hex matches one assigned to **c-primary-alt** ($c-berkshires-green: #388557;).
   <img width="970" alt="solid" src="https://user-images.githubusercontent.com/9633303/67949508-36027700-fbbe-11e9-89aa-456bd836ce32.png">
5. Go to /patternlab/styleguide/source/_patterns/05-pages/marketing-campaign.json
6. Add `"bgImage": "../../assets/images/placeholder/1920x1080-sign.jpg",` after L. 80.
7. Go to the test page and check the key message right above the footer.
4. Find it has a background image with green overlay.  The green color hex matches one assigned to **c-primary-alt-dark** ($c-berkshires-green-dark: #214F34;).
   <img width="969" alt="overlay" src="https://user-images.githubusercontent.com/9633303/67949840-de184000-fbbe-11e9-8988-2ea49ef41322.png">




## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
